doc/releases/migration-guide-4.0.rst

```diff
@@ -288,5 +288,12 @@ Shell
 * ``kernel threads`` and ``kernel stacks`` shell command have been renamed to
   ``kernel thread list`` & ``kernel thread stacks``
 
+JWT (JSON Web Token)
+====================
+
+* By default signature is now computed through PSA Crypto API for both RSA and ECDSA.
+  The newly-added :kconfig:option:`CONFIG_JWT_USE_LEGACY` can be used to switch
+  back to previous libraries (TinyCrypt for ECDSA and Mbed TLS for RSA). (:github:`78243`)
```

Comment:
Would it make sense to add a note here along the lines of:
```
The conversion to the PSA Crypto API is being done in preparation for the deprecation of TinyCrypt.
```
Not everyone follows the security working groups minutes (nor do they read migration guides).  The thought was to to make it more clear the deprecation is happening in multiple areas.